src/components/graphCard/graphCardContext.js

```diff
@@ -145,10 +147,58 @@ const useGetMetrics = ({
   return response;
 };
 
+/**
+ * Return a component list for a configurable graphCard action toolbar
+ *
+ * @param {object} params
+ * @param {Array} params.categoryOptions
+ * @param {Function} params.useMetricsSelector
+ * @param {Function} params.useGraphCardContext
+ * @returns {*[]}
+ */
+const useGraphCardActions = ({
+  categoryOptions = toolbarFieldOptions,
+  useMetricsSelector: useAliasMetricsSelector = useMetricsSelector,
+  useGraphCardContext: useAliasGraphCardContext = useGraphCardContext
+} = {}) => {
+  const [updatedActions, setUpdatedActions] = useState();
+  const { pending, dataSets } = useAliasMetricsSelector();
+  const { settings = {} } = useAliasGraphCardContext();
+  const { actionDisplay, actions = [] } = settings;
+
+  useShallowCompareEffect(() => {
+    if (!pending && dataSets) {
```

Comment:
required: review the pending and dataSets condition

something similar was removed from the metrics call in #1023

Reply:
* converted to useMemo since we're dealing with settings
* pending and dataSets length used to limit "content" callbacks. 
   * in the future we may expand this to include a `typeof content === function` check. The limitation associated with using dataSets is the callback